utils.py

```diff
@@ -286,6 +286,12 @@ def __pow__(self, other):
         return r
 # TODO: Concatenate tf objects into MIMO structure
 
+def RHPonly(x):
+    RHPx = []
+    for i in range(len(x)):
+        if x[i].real > 0:
+            RHPx.append(numpy.round(x[i], 2))
+    return list(set(RHPx))
```

Comment:
I would write this as `return list(set(numpy.round(xi, 2) for xi in x if xi.real > 0))` and probably make the precision on the round an option with a default.